web4/public_html/index.html

```diff
@@ -384,8 +509,9 @@ <h6> ${name} </h6>
     const instanceAccount = location.host.split(".")[0];
     viewer.setAttribute("src", `${instanceAccount}.near/widget/app`);
   } else if (location.port === "8080") {
-    viewer.setAttribute("rpc", "http://127.0.0.1:8080/api/proxy-rpc");
+    //viewer.setAttribute("rpc", "http://127.0.0.1:8080/api/proxy-rpc");
```

Comment:
Why do we comment the RPC proxy?

Reply:
It should not be commented. This was just while developing I think. Thanks for pointing it out :)